apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,306 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <%- include('../partials/head'); %>
+</head>
+<body>
+  <script>
+    $(function () {
+      $('[data-toggle="popover"]').popover({ sanitize: false,   container: 'body' });
+
+      $('#add_new_override').on('submit', function (event) {
+        var startDate = new Date($('#start_date').val());
+        var endDate = new Date($('#end_date').val());
+      
+        if (startDate >= endDate) {
+          event.preventDefault();
+          $('#end_date').addClass('is-invalid');
+          $('#end_date_error').text('End date should be greater than the start date.').show();
+        }
+        
+        
+      });
+      $('.edit-override-button').on('click', function (event) {
+        event.preventDefault();
+        var button = $(this);
+        var editOverrideForm = $('#edit-override-form');
+        editOverrideForm.find('#edit-assessment_id').val(button.data('assessment-id'));
+        editOverrideForm.find('#edit-student_uid').val(button.data('user-id'));
+        editOverrideForm.find('#edit-group_id').val(button.data('group-id'));
+        editOverrideForm.find('#edit-credit').val(button.data('credit')); 
+        var policy_start_date_string = button.data('start-date').slice(0,-6).replace(" ", "T");  
+        var policy_end_date_string = button.data('end-date').slice(0,-6).replace(" ", "T");
```

Comment:
It seems like this is here to account for the fact that these values are strings like `2024-05-24 15:52:22 (CDT)`. Is that correct? This is brittle and won't correctly handle all time zones as some have 4-character names like "AKDT".

I think handling this correctly is outside the realm of what we can do easily with plain, vanilla JavaScript. What we _want_ to do is format this on the server such that we can directly pass it to the `value` of the `datetime-local` input. I think this is best done via something like the [`Temporal` API](https://tc39.es/proposal-temporal/docs/). There is no prior art in PrairieLearn, though we've done something similar in PrairieTest. I'll propose we do one of two things:

- I take over this branch and add the necessary stuff for you.
- You tell me that you'd like to keep working on this, in which case I'll open another PR introducing the function that you'll need to use. Once that's merged, you can use it here.

Let me know which you'd prefer!

Reply:
You can take over the branch. I really wanted to finish but unfortunately I have started my summer internship and have way less time to develop :(

Reply:
🙏 I'll take it from here. Thanks for all the work you've put in so far! I hope this has been a good experience for you. Best of luck with your internship!!

Reply:
This has been resolved in recent commits.